src/utils/normalizer.js

```diff
@@ -1,4 +1,9 @@
-import { __DEV__ } from 'react-native';
+// @flow
+import {
+  FOUR_INCHES,
+  FOUR_DOT_SEVEN_INCHES,
+  FIVE_DOT_FIVE_INCHES,
+} from '../global/Constants';
 
 const React = require('react-native');
```

Comment:
Please use `import` instead of `require`.

Reply:
Done